pkg/api/router.go

```diff
@@ -54,6 +54,10 @@ func (s *server) setupRouting() {
 		"POST": http.HandlerFunc(s.chunkUploadHandler),
 	})
 
+	handle(router, "/bzz:/{address}/{path:.*}", jsonhttp.MethodHandler{
```

Comment:
we agreed to use `/bzz:/` notation?
https://beehive.ethswarm.org/swarm/pl/fzgc13trxpgjz8k37ft3y8zzgo

Reply:
@zelig do you have an opinion on this maybe? @vandot asked if it would be better to use `/bzz/` instead: https://beehive.ethswarm.org/swarm/pl/fzgc13trxpgjz8k37ft3y8zzgo.

Reply:
@vandot Sorry, I did not see that, I was following https://hackmd.io/YklWpLHPRDGWipoZavw50Q, and Book of Swarm: manifest API definition, page 227, section 10.2.4 (+).